src/lexers/jsx-lexer.js

```diff
@@ -34,18 +34,23 @@ export default class JsxLexer extends HTMLLexer {
     return this.keys
   }
 
+  /**
+  * Extract tags and content from the Trans component.
+  * @param {string} string
+  * @returns {array} Array of key options
+  */
   extractTrans(content) {
     let matches
-    const closingTagPattern = '(?:<Trans([^>]*\\s' + this.attr + '[^>]*?)\\/>)'
-    const selfClosingTagPattern = '(?:<Trans([^>]*\\s' + this.attr + '[^>]*?)>((?:\\s|.)*?)<\\/Trans>)'
+    const selfClosingTagPattern = '(?:<\\s*Trans([^>]*)?/>)'
+    const closingTagPattern = '(?:<\\s*Trans([^>]*)?>((?:(?!</\\s*Trans\\s*>)[^])*)</\\s*Trans\\s*>)'
```

Comment:
1. The `\\s*` are places where any amount of space can syntactically occur.
2. Removed strict checking for `this.attr` b/c parseAttributes does that already.
3. Renamed `closingTagPattern` to `selfClosingTagPattern` and vice versa (they were backwards.)